tasks/Prisoner's Dilemma (various versions).md

```diff
@@ -0,0 +1,66 @@
+# Summary
+The N-person prisoners dilemma game is designed to mimic a situation where two prisoners were caught for jointly committing a felony. The two prisoners are held in separate rooms so that they cannot communicate.  The District Attorney cannot prove their guilt but offers each prisoner a plea deal.  If one confesses, they get to go free, but the other receives the maximum sentence.  If they both confess they receive a moderate sentence each.  If neither confess, both receive a minimum sentence.  
+
+# References
+Dawes, Robyn M. "Social dilemmas." Annual review of psychology 31.1 (1980): 169-193.  
+Link: https://www.annualreviews.org/doi/pdf/10.1146/annurev.ps.31.020180.001125?casa_token=Htyc-BVJvIYAAAAA:HKfLuV3zEzJACz32bcDryhwOwqJQThra3Tn7B2iHjdYX5_Ws-xFNnagPOkyvaTL7y1Uu1EJLkcyn
+
+# Stimuli
+## The visual components
+N/A
+
+## Materials for alternative versions of the experiment 
+N/A
+
+# Procedure
+## Steps
+The “Take Some” Game
+- Groups of three are formed.
+- Players are briefed on the rules.
+- Simultaneously each participant holds up a the blue or red poker chip.
+    - For each red chip held up, the player holding the chip is awarded $3.00, but every participant in the group is penalized $1.00 (meaning if each participant holds up a red chip, the groups payout is 0).
+    - For each blue chip held up, the participant holding it is awarded $1.00, and no one is penalized.
+- Scores are tallied, and this can be repeated for as many rounds as the experimenter chooses.
+The “Give Some” Game
+- Participants are broken up into groups of five.
+- Players are briefed on the rules.
+- Each player decides whether to keep $8.00 from the experimenter, or give $3.00 to every other participant in the group.
+- This is repeated for as many rounds as the experimenter chooses.
+General Prisonniers Dilemma
+- Generally played with two people, and can be played repeatedly.
+- Designed to model a scenario where where two prisoners were caught for jointly committing a felony.
+- The two prisoners are held in separate rooms so that they cannot communicate.  The District Attorney cannot prove their guilt but offers each prisoner a plea deal.  If one confesses, they get to go free, but the other receives the maximum sentence.  If they both confess they receive a moderate sentence each.  If neither confess, both receive a minimum sentence.
+
+## Roles 
+Participants work independently within their groups, they can choose to be cooperative or act selfishly.
```

Comment:
I will probably add the description for the two roles, something like:

- Defectors:
Participants choose the dominating strategy of the game, which is to defect. They receive a higher payoff than the cooperators in the game and the highest individual payoff if no other participant defects.
- Cooperators:
Participants choose to cooperate in order to achieve universal cooperation. They will most likely receive a lower payoff than defectors if there is one.